src/stratigraphy/main.py

```diff
@@ -234,6 +248,17 @@ def start_pipeline(
             temp_directory / "document_level_metrics.csv"
         )  # mlflow.log_artifact expects a file
 
+        # print the metrics
+        logger.info("Performance metrics:")
+        logger.info(metrics)
+        logger.info("Groundwater level metrics:")
+        logger.info("Accuracy:")
+        logger.info(round(metrics["groundwater_accuracy"], 3))
+        logger.info("TP:")
+        logger.info(metrics["groundwater_tp"])
+        logger.info("Coordinate metrics:")
+        logger.info(round(metrics["coordinate_accuracy"], 3))
```

Comment:
We need to agree on what we output in the logger and how we format it, because this is starting to become rather messy.

Reply:
I would say we can print the performance metrics.